recipes/quadprog/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "quadprog" %}
+{% set version = "0.1.7" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: f308b9a4f0368c956fd48e480dfadad7a9497ed7c1bb3059a2777b7deb4fbff9
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - cython
+    - pip
+    - python
+  run:
+    - cython
+    - python
+    - numpy
```

Comment:
could you please double-check this dependency here ``numpy``? if the project needs it

Reply:
Thank you for the suggestions @marcelotrevisani! I can confirm `numpy` is a dependency